File: src/main/java/org/robotykhl/minigamesVoting/CommandTab.java
package org.robotykhl.minigamesVoting;

import java.util.List;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.command.TabCompleter;

public class CommandTab implements TabCompleter {

  @Override
  public List<String> onTabComplete(CommandSender sender, Command cmd, String s, String[] args) {
    if (args.length == 1) {
      return List.of("Secconds To Wait");
    } else {
      return List.of();
    }
  }
}

File: src/main/kotlin/io/robotykhl/minigames/CommandTab.kt
package io.robotykhl.minigames

import org.bukkit.command.Command
import org.bukkit.command.CommandSender
import org.bukkit.command.TabCompleter

class CommandTab : TabCompleter {

    override fun onTabComplete(
        sender: CommandSender,
        cmd: Command,
        s: String,
        args: Array<String>
    ): List<String> {
        return if (args.size == 1) {
            listOf("Secconds To Wait")
        } else {
            emptyList()
        }
    }
}